.gitignore

```diff
@@ -33,3 +33,4 @@
 npm-debug.log
 testem.log
 /.chrome
+yarn.lock
```

Comment:
I thought that the `yarn.lock` was was _supposed_ to be checked into source control?
https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control